util/string_test.cpp

```diff
@@ -25,5 +29,30 @@ int main() {
         expect(!no_case_compare("word "sv, "woord"sv));
     });
 
+    etest::test("trim start", [] {
+        expect_eq(trim_start(" abc "), "abc ");
+        expect_eq(trim_start("\t431\r\n"), "431\r\n");
+        expect_eq(trim_start("  hello world!"), "hello world!");
+        expect_eq(trim_start("word "), "word ");
+        expect_eq(trim_start("\r\n"), "");
+    });
+
+    etest::test("trim end", [] {
+        expect_eq(trim_end("abc "), "abc");
+        expect_eq(trim_end("53 \r\n"), "53");
+        expect_eq(trim_end("hello world!\t"), "hello world!");
+        expect_eq(trim_end(" word"), " word");
+        expect_eq(trim_start("\r\n"), "");
```

Comment:
Should be calling `trim_end` and not `trim_start`?

Reply:
That was sloppy! Good catch!